commit 8d07efbdf84f47b935c8bb4e5998050375295764
Author: Geert Uytterhoeven <geert@linux-m68k.org>
Date:   Thu Jun 7 15:50:48 2018 +0200

    ASoC: rt1305: Use ULL suffixes for 64-bit constants
    
    [ Upstream commit 4f29b663c08d369fe320a148179996c94cf7d01b ]
    
    With gcc 4.1.2:
    
        sound/soc/codecs/rt1305.c: In function ‘rt1305_calibrate’:
        sound/soc/codecs/rt1305.c:1069: warning: integer constant is too large for ‘long’ type
        sound/soc/codecs/rt1305.c:1086: warning: integer constant is too large for ‘long’ type
    
    Add the missing "ULL" suffixes to fix this.
    
    Fixes: 29bc643ddd7efb74 ("ASoC: rt1305: Add RT1305/RT1306 amplifier driver")
    Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
    Signed-off-by: Mark Brown <broonie@kernel.org>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/sound/soc/codecs/rt1305.c b/sound/soc/codecs/rt1305.c
index f4c8c45f4010..421b8fb2fa04 100644
--- a/sound/soc/codecs/rt1305.c
+++ b/sound/soc/codecs/rt1305.c
@@ -1066,7 +1066,7 @@ static void rt1305_calibrate(struct rt1305_priv *rt1305)
 	pr_debug("Left_rhl = 0x%x rh=0x%x rl=0x%x\n", rhl, rh, rl);
 	pr_info("Left channel %d.%dohm\n", (r0ohm/10), (r0ohm%10));
 
-	r0l = 562949953421312;
+	r0l = 562949953421312ULL;
 	if (rhl != 0)
 		do_div(r0l, rhl);
 	pr_debug("Left_r0 = 0x%llx\n", r0l);
@@ -1083,7 +1083,7 @@ static void rt1305_calibrate(struct rt1305_priv *rt1305)
 	pr_debug("Right_rhl = 0x%x rh=0x%x rl=0x%x\n", rhl, rh, rl);
 	pr_info("Right channel %d.%dohm\n", (r0ohm/10), (r0ohm%10));
 
-	r0r = 562949953421312;
+	r0r = 562949953421312ULL;
 	if (rhl != 0)
 		do_div(r0r, rhl);
 	pr_debug("Right_r0 = 0x%llx\n", r0r);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1199_8d07efbdf84f47b935c8bb4e5998050375295764.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1199_8d07efbdf84f47b935c8bb4e5998050375295764.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
